src/components/Icon.js

```diff
@@ -16,7 +16,7 @@ export const Icon = ({ xml, width, height, name, size, focused, iconColor, style
   const color = iconColor || (focused ? colors.accent : colors.primary);
 
   return (
-    <View style={style}>
+    <View style={style} hitSlop={{ top: 12, bottom: 12 }}>
```

Comment:
does this work for everywhere we use `Icon`? or are there maybe overlapping click areas somewhere?

Reply:
the result of my tests is positive, meaning on the smallest screen (at my disposal) iPhone SE, there are no overlaps. However, I have problems in making tests using the secrets of other cities, I cannot see some sections, such as the service area on the home screen, which is essential for tests this new feature . In more simple words if use Bad Belzig secrets I don't see the same public app , I can't see Bad Belzig data but for example service area is not visible. 
What am I doing wrong ?

Reply:
don't know from that description. changing secrets should work live and direct and the app refreshes a few times.
if not, stop yarn, change secrets, start yarn.
pull to refresh can also be a fix, when it seems to show old data.

Reply:
sorry it is a really bad description .

Reply:
your or mine? :D can i do or tell something more?

Reply:
> the result of my tests is positive, meaning on the smallest screen (at my disposal) iPhone SE, there are no overlaps. However, I have problems using the secrets of different cities, I cannot see some sections, such as the service area on the home screen, which is essential for tests this new feature . In more simple words if use Bad Belzig secrets I don't see the same public app , I can see Bad Belzig data but for example service area is not visible.
> What am I doing wrong ?

mine was horrible... ! 

I think it time to merge this branch , what do you think ? :D

Reply:
ok